x/tokenregistry/types/msgs.go

```diff
@@ -105,3 +109,69 @@ func (m *MsgDeregister) GetSigners() []sdk.AccAddress {
 
 	return []sdk.AccAddress{addr}
 }
+
+// NewTokenMetadata is a constructor function for MsgTokenMetadataAdd
+func NewTokenMetadata(cosmosSender sdk.AccAddress,
```

Comment:
A function returning TokenMetadataAddRequest shouldn't be named NewTokenMetadata

Reply:
will change the method name